calabash-cucumber/lib/calabash-cucumber/launcher.rb

```diff
@@ -787,9 +787,6 @@ def ensure_connectivity(max_retry=10, timeout=30)
               begin
                 connected = (ping_app == '200')
                 break if connected
-              rescue Exception => e
```

Comment:
@TobiasRoikjer I think that all I want to do is rescue the CalabashLauncherTimeoutErr.